.eslintignore

```diff
@@ -0,0 +1,2 @@
+dist
+jest.config.*
```

Comment:
Looking at the diff I suggest `.editorconfig` with `insert_final_newline = true`

Reply:
> The reason you need at least one newline is that historically some tools have problems if the file ends and the last line has text on it but does not have a newline at the end of the file. Badly written tools can miss processing that last partial line, or even worse can read random memory beyond the last line (though that is unlikely to happen with tools written in Python it can happen with tools written in C).

https://stackoverflow.com/questions/43491498/python-script-should-end-with-new-line-or-not-pylint-contradicting-itself/43491581#43491581

Reply:
that doesn't seem to do anything on vscode?